### PR TITLE
Fixed minor html compliance issues found via SonarQube

### PR DESCRIPTION
**Pull request by TimYagan**
Fixed minor html compliance issues found via SonarQube - Low Risk and improves browser compatibility

### CODE REVIEW

File: grouper/fe/templates/email/base.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html>
 
 <head>
+    <<title></title>
```
Comment (rra) [inline grouper/fe/templates/email/base.html]: Extra open `<` here, I think?

File: grouper/fe/templates/macros/ui.html [hunk 2]
```diff
@@ -303,18 +303,18 @@ <h3 class="panel-title">Service Accounts</h3>
 {% macro account(user, type=None) -%}
 {% if type == None %}
 <a class="account-link" href="/{{ user.type | lower }}s/{{ user.name }}">
-    <i class="fa fa-user{% if user.type.lower() == 'group' %}s{% endif %}"></i> 
+    <em class="fa fa-user{% if user.type.lower() == 'group' %}s{% endif %}"></em> 
     {% if user.role_user or user.is_service_account %}
     <i>
```
Comment (rra) [inline grouper/fe/templates/macros/ui.html]: Need to change this one as well.

File: grouper/fe/templates/service.html [hunk 3]
```diff
@@ -147,7 +147,7 @@ <h4 class="modal-title">Enable User</h4>
                 <div class="modal-body">
                     <p>Are you sure you want to enable this user?</p>
                     <label class="checkbox normal-weight">
-                        <i>preserve any pre-existing group membership (you don't want this)</i>
+                        <i>preserve any pre-existing group membership (you don't want this)</em>
```
Comment (rra) [inline grouper/fe/templates/service.html]: Also need to change the `<i>`.

**Review by rra — CHANGES_REQUESTED** [review]
I'm dubious this will improve browser compatibility all that much, but agreed with low risk and happy to take the change anyway since it makes automated tools happier. (In reality much of this should be in style sheets instead, but that's a much larger change.)

There were just a few places where I noticed a few problems, and then I'm happy to merge this.

Thanks for your contribution!

**Comment by TimYagan** [comment]
Thanks for the review. I wilö change that.

**Pull request closed without merging**